src/Microsoft.Azure.SignalR.Management/RestHubLifetimeManager.cs

```diff
@@ -196,6 +198,28 @@ public Task UserRemoveFromAllGroupsAsync(string userId, CancellationToken cancel
             return _restClient.SendAsync(api, HttpMethod.Delete, _productInfo, cancellationToken: cancellationToken);
         }
 
+        public async Task<bool> IsUserInGroup(string userId, string groupName, CancellationToken cancellationToken = default)
+        {
+            var isUserInGroup = false;
+            var api = _restApiProvider.GetUserGroupManagementEndpoint(_appName, _hubName, userId, groupName);
+            await _restClient.SendAsync(api, HttpMethod.Get, _productInfo, handleResponse: (request, response) =>
+                {
+                    switch (response.StatusCode)
+                    {
+                        case HttpStatusCode.OK:
+                            isUserInGroup = true;
+                            break;
+                        case HttpStatusCode.NotFound:
+                            break;
+                        default:
+                            var innerException = _restClient.GenerateInnerExceptionOnResponseFailure(response.StatusCode, response.ReasonPhrase);
+                            throw new AzureSignalRRuntimeException(response.RequestMessage.RequestUri.ToString(), innerException);
```

Comment:
what about 401 unauthorized, 400 bad request?

Reply:
Will throw in next commit....

Reply:
Fixed.